### ISSUE TITLE
[request] Change core option

### ISSUE THREAD

**Issue by lollo78** [libretro/Lakka#273]
Please, include an option into the retroarch Quick Menu to "Change core".
I select the wrong core for some games and the only way to reselect the right core from the GUI is to manually load it from Main Menu/Load Content/Select File And Detect Core.
Thx

**Comment by Kivutar** [comment]
Sorry, but I would like to limit the number of entries in the quick menu, which is already too bloated for my taste..

**Comment by lollo78** [comment]
Please, don't close this request.
Consider to add the voice "Change core" in this retroarch  menu:
![1457641847993690391554](https://cloud.githubusercontent.com/assets/6298682/13683528/6f369260-e707-11e5-8a00-a1757fb0b898.jpg)
At the moment, if we select the wrong core (the first time on game launch) we have to manually delete the bad core association inside the playlist file. 
But consider that this feature is also important if anyone want to try another core with a specific rom (some rom perform better with a specific core, instead of the most popular core for the same platform).

**Comment by pokemontrainerAsh** [comment]
As a normal user, I selected a broken core and I find it ridiculous that there's no way to change the core. I find a post telling me to edit the playlist file. Well, I broke the whole thing. There needs to be an intuitive way to change the core when you select the game. The way it's setup now, you only get "one chance" to select the right core. Also, shouldn't the core be tied to a system and not a particular game?

**Comment by Kivutar** [comment]
You're right. I reopen this. And @fr500 found a place to do that cleanly.

**Comment by hizzlekizzle** [comment]
You can change the core assigned to a playlist by going to settings > playlist and changing it there.

**Comment by Kivutar** [comment]
Yeah but they are talking of changing the core assigned to a given playlist entry.

**Comment by daviderickson** [comment]
Yes please add this!!  It is super frustrating to not be able to change the core assigned to a game from the GUI...

**Comment by manekinekodesu** [comment]
Still, not implemented in Lakka 2.1 RC1! Very frustrating issue. An option in "Run" menu would fix this:
\> Run
\> Run with...
\> Remove
\> Information

**Comment by SiTWulf** [comment]
Not news for now...
the only option to fix this is edit the playlist and change the core to DETECT to reselect a new core.

**Comment by D3miS3raph** [comment]
Still hoping for an option to fix this or a button to set a rom entry back to DETECT without doing it manually.  You could code an option after you click on a rom, that in the sub menu "reset to Detect mode", you click it, and it edits the entry.  Or any number of other ways.  Very hard to play with multiple cores right now :(
